source/site-administrators/studio/using-the-proxy-config.rst

```diff
@@ -0,0 +1,161 @@
+
+-------
+Example
+-------
+
+We'll look at the video center blueprint, a React application available from the public marketplace, that runs on ``localhost:3000``, then setup the Studio proxy so we can preview the React application inside Studio.  Finally, we'll make some changes in the React application and view the changes made inside Studio.
+
+Let's begin:
+
+#. Setup the React application.
+
+   * Clone the video center blueprint by running ``git clone https://github.com/craftercms/video-center-blueprint.git``
+
+       .. code-block:: sh
+
+          ➜ git clone https://github.com/craftercms/video-center-blueprint.git
+          Cloning into 'video-center-blueprint'...
+          remote: Enumerating objects: 6433, done.
+          remote: Total 6433 (delta 0), reused 0 (delta 0), pack-reused 6433
+          Receiving objects: 100% (6433/6433), 77.12 MiB | 4.92 MiB/s, done.
+          Resolving deltas: 100% (4041/4041), done.
+
+   * Run the React application
+
+     Inside the video center blueprint folder that we just cloned above, navigate to ``video-center-blueprint/sources/app``.
+     We need ``yarn`` installed in your system.  Running ``yarn`` with no command will run ``yarn install``.  In the example below, yarn is already installed in the system
+
+       .. code-block:: sh
+
+          ➜  app git:(master) yarn
+          yarn install v1.22.4
+          [1/4] 🔍  Resolving packages...
+          success Already up-to-date.
+          ✨  Done in 0.68s.
+
+     Build the React application by running ``yarn start``
+
+       .. code-block:: sh
+
+          ➜  app git:(master) yarn start
+          yarn run v1.22.4
+          $react-scripts start
+
+          Compiled successfully!
+
+          You can now view video-center-blueprint in the browser.
+
+          Local:            http://localhost:3000/
+          On Your Network:  http://192.168.1.135:3000/
+
+          Note that the development build is not optimized.
+          To create a production build, use yarn build.
+
+     The command above will open a browser window where we can view the app
+
+     .. image:: /_static/images/site-admin/vcbp-preview.jpg
+        :alt: Video Center Blueprint preview on "localhost:3000"
+        :width: 70 %
+        :align: center
+
+     |
+
+   * Edit the base url in your React app to the server we are using for development, which in our case, is the Studio server, on ``localhost:8080``, and the websocket port for hot module reloading, then save the changes and restart the React app.
+
+     To edit the base url, navigate to ``video-center-blueprint/sources/app`` then open the ``.env.development`` file using your favorite editor and set the following variables variables: ``REACT_APP_BASE_URL`` and ``WDS_SOCKET_PORT``
+
+       .. code-block:: text
+          :caption: *video-center-blueprint/sources/app/.env.development*
+
+          REACT_APP_BASE_URL=http://localhost:8080
```

Comment:
I think we need to mention that the react server needs to be restarted for this to work right?